content/guidance/apimanagement/index.html.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: DRAFT API Management Guidance
+title: API Management Guidance
```

Comment:
Did we want to, in each page's generated content, make it clearer they're a draft, as we had before, or are we happy that the status being draft is sufficient?

Reply:
In the current version it doesn't have draft in the contents page title, which is the only place this is used at the moment - what do you think @charlesbaird-dsa?

Reply:
I was thinking also in the header for i.e. https://alphagov.github.io/data-standards-authority/guidance/apimanagement/#draft-api-management-guidance-draft and whether we want to make it clearer in the title, or whether the note below is sufficient

Reply:
ah I see, yeah good point

Reply:
This is probably a larger conversation tbh - when the github version is the same as what's on GOV.UK and we're not actively working on it at the moment, there's a bit of a question mark as to whether that's actually a draft (it's arguably a new draft with no changes on it yet). 

It might be something like: 

- brand new guidance still in draft format (doesn't yet exist on GOV.UK) - title says DRAFT, status is `draft`
- live guidance with a working draft on github - title is plain, status is `draft`
- live guidance with an untouched draft on github - title is plain, status is `live`

But as I said, this is probably a bigger conversation!

Reply:
Good shout, raised https://github.com/alphagov/data-standards-authority/issues/98 for discussion :+1: